web/spacewalk-web.changes

```diff
@@ -1,3 +1,6 @@
+- Change the order of the Refresh and Create buttons on the
+  image-profiles and image-stores pages to stay consistent
+  with the image-list page.
```

Comment:
It would be better to make this shorter. How about something like:
```
Align the order of the Refresh and Create buttons for more consistency
```

Reply:
Yes, it sounds better, it's updated now